src/wrapper/theta.rs

```diff
@@ -133,8 +133,11 @@ pub struct ThetaIntersection {
 }
 
 impl ThetaIntersection {
-    /// Create a theta union over nothing, which corresponds to the
-    /// empty set.
+    /// Create a theta intersection over nothing, which corresponds to
+    /// an infinite set.
```

Comment:
"a universal set containing all items, represented as None in `.sketch()`" (not all infinite sets have every key)